### PR TITLE
[7] Designed a Contact form using HTML

### PR DESCRIPTION
**Pull request by Iyiola-stack**
## Task: `Contact form
- `Changes you made to given task`
Contactform

### CODE REVIEW

File: submissions/peterIyiola/html/productPage-1.html [hunk 1]
```diff
@@ -1,3 +1,8 @@
+<<<<<<< HEAD
```
Comment (ToluAkin) [inline submissions/peterIyiola/html/productPage-1.html:1]: You are only required to add a contact form page, the other files added are not needed.

File: submissions/peterIyiola/html/contactForm.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
+
```
Comment (ToluAkin) [inline submissions/peterIyiola/html/contactForm.html:1]: Remove extra the lines here.

Comment (Iyiola-stack) [reply submissions/peterIyiola/html/contactForm.html]: Done

File: submissions/peterIyiola/html/productPage-1.html [hunk 3]
```diff
@@ -6,6 +6,11 @@
    </head>
    <body>
       <div>
+         <h1><b>Better Buys</b></h1>
+         <h4><i>Your neighbourhood electronics store</i></h4>
+         <hr>
+         <a href="C:\Users\HP\Desktop\Dufuna-CodeCamp20\submissions\peterIyiola\htmlForms\SignUp.html" target="_blank">Signup page</a><br><br>
```
Comment (ToluAkin) [inline submissions/peterIyiola/html/productPage-1.html:12]: @Iyiola-stack Use relative path to link your images and pages.

File: submissions/peterIyiola/htmlForms/SignUp.html [hunk 4]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8">
+      <title>Signup</title>
+   </head>
+   <body>
+      <h1>Better Buys' Signup Page</h1>
+      <a href="C:\Users\HP\Desktop\Dufuna-CodeCamp20\submissions\peterIyiola\html\productPage-1.html" target="_blank">Product page</a><br><br>
```
Comment (ToluAkin) [inline submissions/peterIyiola/htmlForms/SignUp.html:9]: Same as above

File: submissions/peterIyiola/htmlForms/SignUp.html [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+      <a href="C:\Users\HP\Desktop\Dufuna-CodeCamp20\submissions\peterIyiola\htmlForms\contactForm.html" target="_blank">Contact Form</a><br>
```
Comment (ToluAkin) [inline submissions/peterIyiola/htmlForms/SignUp.html:10]: Same as above

**Pull request closed without merging**